pkg/k8s/informer/benchmarks/informer_benchmarks_test.go

```diff
@@ -56,6 +56,8 @@ var nodeSampleJSON = `{
             "cloud.google.com/gke-nodepool": "default-pool",
             "cloud.google.com/gke-os-distribution": "cos",
             "disktype": "ssd",
+            "topology.kubernetes.io/region": "earth",
+            "topology.kubernetes.io/zone": "earth",
             "failure-domain.beta.kubernetes.io/region": "earth",
```

Comment:
can you add an inline comment to remove after support for 1.17 is dropped as well?

Reply:
Updated. Sorry for the delay.